### DISCUSSION TITLE
Domino online meeting integration behind firewall

### DISCUSSION THREAD

**Discussion by Sistem2021** [HCL-TECH-SOFTWARE/domino-online-meeting-integration#152] [Q&A]
Does anyone know which Urls are needed in the firewall as an exception for Domi to work with TEAMS?
We have these urls allowed: https://docs.microsoft.com/en-us/microsoft-365/enterprise/urls-and-ip-address-ranges?view=o365-worldwide#skype-for-business-online-and -microsoft-teams

Allowing this in the firewall does not work but if we allow all network traffic it works correctly!! 
Any Idea ? thanks in advance ...

**Comment by paulswithers** [comment]
There are URLs used for retrieving tokens in domiConstantsBE.lss Script Library. These might need adding.
